tests/integration/test_wazuh_db/test_wazuh_db.py

```diff
@@ -156,23 +216,29 @@ def restart_wazuh(request):
 
 
 def execute_wazuh_db_query(command):
-    """Function to send a command to the wazuh-db socket.
+    '''
+    Function to send a command to the wazuh-db socket.
+
     Args:
         command(str): Message to send to the socket.
+
     Returns:
         str: A response from the socket
-    """
+    '''
```

Comment:
I would not insert a new line before the one-line summary and use `"""` instead.

Reply:
Done in the following [commit](https://github.com/wazuh/wazuh-qa/commit/5ceb7bd05d809dc92ccde0df5d314a5914036df8)